dotnet/src/Connectors/Connectors.Memory.Sqlite/Conditions/SqliteWhereCondition.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Connectors.Sqlite;
+
+internal abstract record SqliteWhereCondition(string Operand, List<object> Values, string? TableName = null)
```

Comment:
My understanding was that we were advised to avoid record because of the overhead involved in using them?